pandas/tests/types/test_validate.py

```diff
@@ -0,0 +1,169 @@
+from pandas.types.validate import _validate_bool_type
+from unittest import TestCase
```

Comment:
you can put the categorical tests in tests_categorical. iow, these tests go with the type that they are testing.